tick.go

```diff
@@ -124,12 +124,13 @@ func GeneratePrettyContinuousTicks(r Renderer, ra Range, isVertical bool, style
 	}
 
 	prettyStepsPriorityList := []float64{1, 5, 2, 2.5, 4, 3}
-	paramWeights := map[string]float64{
-		"simplicity": 0.2,
-		"coverage":   0.25,
-		"density":    0.5,
-		"legibility": 0.05,
-	}
+
+	var (
```

Comment:
Maybe we should use `const` instead of `var` since the values are never changed throughout the program

Reply:
yes, chaned it. thx